majasdarbs/src/io/codelex/arrays/practice/TicTacToe.java

```diff
@@ -0,0 +1,122 @@
+package io.codelex.arrays.practice;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+    private static final char[][] board = new char[3][3];
+
+
+    public static void main(String[] args) {
+
+        initBoard();
+
+        while (!checkWinner(board)) {
+
+            displayBoard();
+
+            game(board, "Player1");
+
+            displayBoard();
+
+            game(board, "Player2");
+
+
+        }
+
+    }
+
+
+    public static void initBoard() {
+        for (int r = 0; r < 3; r++) {
+            for (int c = 0; c < 3; c++) {
+                board[r][c] = ' ';
+            }
+        }
+    }
+
+    public static void displayBoard() {
+        System.out.println("  0  " + board[0][0] + "|" + board[0][1] + "|" + board[0][2]);
+        System.out.println("    --+-+--");
+        System.out.println("  1  " + board[1][0] + "|" + board[1][1] + "|" + board[1][2]);
+        System.out.println("    --+-+--");
+        System.out.println("  2  " + board[2][0] + "|" + board[2][1] + "|" + board[2][2]);
+        System.out.println("     0 1 2 ");
+    }
+
+    public static Boolean checkWinner(char[][] board) {
+        boolean check = (board[0][0] != ' ') && (board[0][0] == board[0][1]) && (board[0][1] == board[0][2]);
+
+        if ((board[1][0] != ' ') && (board[1][0] == board[1][1]) && (board[1][1] == board[1][2])) {
+            check = true;
+        }
+        if ((board[2][0] != ' ') && (board[2][0] == board[2][1]) && (board[2][1] == board[2][2])) {
+            check = true;
+        }
+        if ((board[0][0] != ' ') && (board[0][0] == board[1][0]) && (board[1][0] == board[2][0])) {
+            check = true;
+        }
+        if ((board[0][1] != ' ') && (board[0][1] == board[1][1]) && (board[1][1] == board[2][1])) {
+            check = true;
+        }
+        if ((board[0][2] != ' ') && (board[0][2] == board[1][2]) && (board[1][2] == board[2][2])) {
+            check = true;
+        }
+        if ((board[0][0] != ' ') && (board[0][0] == board[1][1]) && (board[1][1] == board[2][2])) {
+            check = true;
+        }
+        if ((board[2][0] != ' ') && (board[2][0] == board[1][1]) && (board[1][1] == board[0][2])) {
+            check = true;
+        }
+
+        return check;
+
+    }
+
+    public static Boolean checkTie(char[][] board) {
+
+        boolean tie = (board[0][0] != ' ') && (board[0][0] == board[0][1]) && (board[0][1] == board[0][2])
+                && (board[0][2] == board[1][0]) && (board[1][0] == board[1][1]) && (board[1][1] == board[1][2])
+                && (board[1][2] == board[2][0]) && (board[2][0] == board[2][1]) && (board[2][1] == board[2][2]);
+
+        return tie;
+    }
+
+    public static void game(char[][] board, String user) {
+
+        char symbol = 'x';
+
```

Comment:
baigi daudz tie new lines, ar vienu toč pietiek :smile:

Reply:
*empty lines

Reply:
aa. paldies :)